collectors/pod.go

```diff
@@ -324,13 +338,22 @@ func (pc *podCollector) collectPod(ch chan<- prometheus.Metric, p v1.Pod) {
 				c.Name, nodeName)
 		}
 
+		if gpu, ok := req[v1.ResourceNvidiaGPU]; ok {
```

Comment:
"Seems like apiserver copies the value setted for limits to requests?"
not seeing the code doing this work from a quick search.
 But I remember long time ago (maybe k8s can use gpu feature just come up)the "GPUs can be specified in the limits section only" hasn't be included in the doc.maybe it is added recently.